exercises/sum-of-multiples/example.py

```diff
@@ -1,5 +1,8 @@
-def sum_of_multiples(limit, multiples):
-    return sum(value for value in range(limit)
-               if any(value % multiple == 0
-                      for multiple in multiples
-                      if multiple > 0))
+old_sum = sum
```

Comment:
This is needed to preserve the sum function in some way, as it is used in the example.